win32ss/gdi/ntgdi/freetype.c

```diff
@@ -334,6 +334,119 @@ SharedFace_Release(PSHARED_FACE Ptr)
     IntUnLockFreeType();
 }
 
+VOID DumpFontGDI(PFONTGDI FontGDI)
+{
+    const char *family_name;
+    const char *style_name;
+
+    if (!FontGDI)
+    {
+        DPRINT("FontGDI NULL\n");
+        return;
+    }
+
+    if (FontGDI->SharedFace)
+    {
+        family_name = FontGDI->SharedFace->Face->family_name;
```

Comment:
Is `SharedFace->Face` always valid?
Might be a good idea to also check this (Since it's a debugging function it should not crash)

Reply:
OK.